README.md

```diff
@@ -4,8 +4,10 @@ Finished app made by following http://guides.railsgirls.com/app
 
 ## Setup
 
-First of all, make sure you have ruby and rails available. Check out [the
-rails girls install guide][rg-install] for information on how to get set up.
+First of all, make sure you have Ruby (programming language) and Ruby on Rails
+(web application framework) available.
+Check out [the rails girls install guide][rg-install] for information on
```

Comment:
Does this actually link anywhere?

Reply:
Yus, there's a variable a few lines down